The Makefile allows installation into other places than /usr/local
and honors standard compiler flags.

Provided by: Sven Joachim <svenjoac@gmx.de>

---
 Makefile |   10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -1,13 +1,17 @@
 # Makefile for symlinks
  
+prefix = /usr/local
+
 all: symlinks
 
+
 symlinks: symlinks.c
-	gcc -Wall -Wstrict-prototypes -O2 -N -s -o symlinks symlinks.c
+	$(CC) -o symlinks $(CPPFLAGS) $(DEFS) $(CFLAGS) symlinks.c $(LDFLAGS)
  
 install: all symlinks.8
-	install -m 755 -o root -g root symlinks /usr/local/bin
-	install -m 644 -o root -g root symlinks.8 /usr/local/man/man8
+	install -d $(prefix)/bin $(prefix)/man/man8
+	install -m 755 -o root -g root symlinks $(prefix)/bin
+	install -m 644 -o root -g root symlinks.8 $(prefix)/man/man8
 
 clean:
 	rm -f symlinks *.o core
